backend/experiment/tests/test_admin_experiment.py

```diff
@@ -246,24 +251,40 @@ def test_related_series_with_series(self):
     def test_blocks_with_no_blocks(self):
         series = Experiment.objects.create(name='Test Series')
         phase = Phase.objects.create(
-            name='Test Group', index=1, randomize=False, dashboard=True, series=series)
+            name='Test Group',
+            index=1,
+            randomize=False,
+            dashboard=True,
+            series=series
+            )
         blocks = self.admin.blocks(phase)
         self.assertEqual(blocks, "No blocks")
 
     def test_blocks_with_blocks(self):
```

Comment:
I guess this test should be called `test_experiment_with_blocks`, and that the first `experiment` was renamed when finding & replacing.

Reply:
Excellent find!